server/plugin.go

```diff
@@ -146,6 +156,33 @@ func (p *Plugin) OnPluginClusterEvent(c *plugin.Context, ev model.PluginClusterE
 	p.HandleClusterEvent(ev)
 }
 
+func (p *Plugin) MessageWillBePosted(c *plugin.Context, post *model.Post) (*model.Post, string) {
+	// If not enabled in config, ignore.
+	if !p.getConfiguration().EnableCodePreview {
+		return nil, ""
+	}
+
+	if post.UserId == "" {
+		return nil, ""
+	}
+
+	msg := post.Message
+	info, err := p.getGitlabUserInfoByMattermostID(post.UserId)
+	if err != nil {
+		p.API.LogError("error in getting user info", "error", err.Message)
```

Comment:
```suggestion
		p.API.LogError("error in getting user info", "error", err.Error())
```

Reply:
It would be `error.Message` only because `getGitlabUserInfoByMattermostID` is returning error of type 
```
type APIErrorResponse struct {
	ID         string `json:"id"`
	Message    string `json:"message"`
	StatusCode int    `json:"status_code"`
}
```